packages/@aws-cdk/aws-fsx/README.md

```diff
@@ -17,8 +17,188 @@
+file system mounted on the instance at start-up:
+
+```ts
+const app = new App();
+const stack = new Stack(app, 'AwsCdkFsxLustre');
+const vpc = new Vpc(stack, 'VPC');
+
+// Create a security group to share across the file system and ec2 instance and open the FSx ports.
+const securityGroup = new SecurityGroup(stack, 'FsxLustreSecurityGroup', {
+  vpc
+});
+securityGroup.addIngressRule(Peer.anyIpv4(), Port.tcp(988), 'Default FSx port ingress');
+securityGroup.addEgressRule(Peer.anyIpv4(), Port.tcp(988), 'Default FSx port egress');
+
+const lustreConfiguration = {
+  deploymentType: FsxLustreDeploymentType.SCRATCH_2,
+};
+const fs = new FsxLustreFileSystem(stack, 'FsxLustreFileSystem', {
+  lustreConfiguration,
+  securityGroup,
+  storageCapacity: 1200,
+  vpc,
+  vpcSubnet: vpc.privateSubnets[0]});
+
+const inst = new Instance(stack, 'Ec2Instance', {
+  instanceType: InstanceType.of(InstanceClass.T2, InstanceSize.LARGE),
+  machineImage: new AmazonLinuxImage({
+    generation: AmazonLinuxGeneration.AMAZON_LINUX_2
+  }),
+  securityGroup,
+  vpc,
+  vpcSubnets: {
+    subnetType: SubnetType.PUBLIC,
+  }
+});
+
+// Need to give the instance access to read information about FSx to determine the file system's mount name.
+inst.role.addManagedPolicy(ManagedPolicy.fromAwsManagedPolicyName("AmazonFSxReadOnlyAccess"));
+
+// Add a dependency on the file system to the instance so that we can be sure the file system is instantiated before querying
+// for its mount name.
+inst.node.addDependency(fs);
+
+const mountPath = '/mnt/fsx';
+const dnsName = `${fs.fileSystemId}.fsx.${stack.region}.amazonaws.com`;
+
+inst.userData.addCommands(
+  'set -eux',
+  'yum update -y',
+  'amazon-linux-extras install -y lustre2.10',
+
+  // The version of the AWS CLI currently on AL2 doesn't surface the required MountName field, so install the latest version.
+  'curl "https://awscli.amazonaws.com/awscli-exe-linux-x86_64.zip" -o "awscliv2.zip"',
+  'unzip awscliv2.zip',
+  './aws/install',
+
+  // Write a python script to parse the json response from the AWS CLI and determine the mount name for the file system.
+  'mountScript=$(mktemp --suffix=".py")',
+  `echo 'import json,sys' >> $mountScript`,
+  `echo 'obj=json.load(sys.stdin)' >> $mountScript`,
+  `echo 'for fs in obj["FileSystems"]:' >> $mountScript`,
+  `echo '  if fs["DNSName"] == "${dnsName}":' >> $mountScript`,
+  `echo '    print(fs["LustreConfiguration"]["MountName"])' >> $mountScript`,
+  `chmod +x $mountScript`,
+
+  // Call the AWS CLI and pipe it into the python script.
+  `mountName=$(/usr/local/bin/aws fsx describe-file-systems --output json --region ${stack.region} | python $mountScript)`,
+
+  // Set up the directory to mount the file system to and mounting it.
+  `mkdir -p ${mountPath}`,
+  `chmod 777 ${mountPath}`,
+  `echo "${dnsName}@tcp:/$mountName ${mountPath} lustre defaults,noatime,flock,_netdev 0 0" | tee -a /etc/fstab`,
+  `mount -t lustre ${dnsName}@tcp:/$mountName ${mountPath}`,
```

Comment:
```
mount -a
```

Will mount everything from `fstab`, iirc. At least there's a command for that, use that instead to remove the duplication.

Reply:
Yes the `-a` flag worked. Fixed.